core/src/cluster_tests.rs

```diff
@@ -137,32 +138,69 @@ pub fn kill_entry_and_spend_and_verify_rest(
         if ingress_node.id == entry_point_info.id {
             continue;
         }
-        let random_keypair = Keypair::new();
+
         let mut client = mk_client(&ingress_node);
         let bal = client
             .poll_get_balance(&funding_keypair.pubkey())
             .expect("balance in source");
         assert!(bal > 0);
-        let mut transaction = SystemTransaction::new_move(
-            &funding_keypair,
-            &random_keypair.pubkey(),
-            1,
-            client.get_recent_blockhash(),
-            0,
-        );
-        let sig = client
-            .retry_transfer(&funding_keypair, &mut transaction, 5)
-            .unwrap();
-        for validator in &cluster_nodes {
-            if validator.id == entry_point_info.id {
-                continue;
+
+        let mut result = Ok(());
+        let mut retries = 0;
+        loop {
+            retries += 1;
+            if retries > 5 {
```

Comment:
hard-coded 5 retries of a `client.retry_transfer` that hard-codes 5 retries?

Reply:
@garious, the problem this is working around is that there is no confirmation count feedback for a signature.  So there is no way to know if it has been replicated yet

Reply:
@garious, the crux of the change is that if poll_all_nodes_for_signature() fails, then we need to also retry the initial `client.retry_transfer` rather than just throwing an error.